_overviews/collections/strings.md

```diff
@@ -26,4 +26,4 @@ Like arrays, strings are not directly sequences, but they can be converted to th
     scala> val s: Seq[Char] = str
     s: Seq[Char] = WrappedString(h, e, l, l, o)
 
-These operations are supported by two implicit conversions. The first, low-priority conversion maps a `String` to a `WrappedString`, which is a subclass of `immutable.IndexedSeq`, This conversion got applied in the last line above where a string got converted into a Seq. the other, high-priority conversion maps a string to a `StringOps` object, which adds all methods on immutable sequences to strings. This conversion was implicitly inserted in the method calls of `reverse`, `map`, `drop`, and `slice` in the example above.
+These operations are supported by two implicit conversions. The first, low-priority conversion maps a `String` to a `WrappedString`, which is a subclass of `immutable.IndexedSeq`, This conversion got applied in the last line above where a string got converted into a Seq. the other, high-priority conversion maps a string to a `StringOps` object, which adds all methods on immutable sequences to string. This conversion was implicitly inserted in the method calls of `reverse`, `map`, `drop`, and `slice` in the example above.
```

Comment:
I think "strings" was fine, but if we're switching to the singular, it would be nice to capitalized and in monospace font, to be clear it refers to the class. Probably better to just go back to "strings".